### DISCUSSION TITLE
The serverless function has crashed

### DISCUSSION THREAD

**Discussion by ManashAnand** [vercel/community#2961] [Help] [answered]
### Summary

I am deploying a full stack web app but everytime it is giving me error for The serverless function has crashed
I don't know why it's happening the same thing is working on my friend laptop but when I try to do it it crashed
Please help 

![suck](https://github.com/vercel/community/assets/91553087/08485e9e-51bf-4fe1-a4d3-d3f8de9efaf4)

### Example

https://vercel.com/new/success?developer-id=&external-id=&redirect-url=&branch=master&deploymentUrl=test-ibuma76pr-manashanand.vercel.app&projectName=test&s=https%3A%2F%2Fgithub.com%2FManashAnand%2Ftest&gitOrgLimit=&hasTrialAvailable=1&totalProjects=1

### Steps to Reproduce

I am deploying a full stack web app but everytime it is giving me error for The serverless function has crashed
I don't know why it's happening the same thing is working on my friend laptop but when I try to do it it crashed
Please help 

![suck](https://github.com/vercel/community/assets/91553087/08485e9e-51bf-4fe1-a4d3-d3f8de9efaf4)

**Comment by amyegan** [comment, marked as answer]
Hello @ManashAnand! You should be able to get more info from the [runtime logs](https://vercel.com/docs/concepts/observability/runtime-logs). 

One common problem is missing or expired environment variables, but there are may other potential causes of unhandled function errors. But you can find more detail in the logs to help you determine exactly what the error is in this case.

**Reply by ManashAnand** [reply]
Hi @amyegan .
I have checked the logs but doesn't exactly got to know what are the problem
I will really apereciate if you can tell me what is wrong with my code or server?
This is my logs 
![e1](https://github.com/vercel/community/assets/91553087/8bf07469-ac0c-4148-a8e9-1a5a4f009d81)

This is my basic code
![e3](https://github.com/vercel/community/assets/91553087/84c47190-b844-421a-bc21-7ae310328026)
![e2](https://github.com/vercel/community/assets/91553087/e07e376e-7317-4bdb-a977-9b596d0c5245)


and I am making sure to put env file in the check box,
So I don't think that will be the issue

**Reply by amyegan** [reply]
If you click on a log row from the first screenshot, then you will get more details about it. It's also a good idea to wrap the function in a [try/catch statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) to help you log specific details to help with debugging when an error occurs.

**Comment by KanikaSharma21** [comment]
did you get the solution @ManashAnand 
I am facing the same issue.
Please help

**Reply by ainuttaqimzawaid** [reply]
i am facing the same issue too, please help us

**Reply by amaroto** [reply]
the same for me :S

**Comment by EdisonMatos** [comment]
Any updates? Same issue here
